lua/core/utils.lua

```diff
@@ -3,9 +3,12 @@ local merge_tb = vim.tbl_deep_extend
 
 M.load_config = function()
   local config = require "core.default_config"
-  local chadrc_exists, chadrc = pcall(require, "custom.chadrc")
+
+  -- nvim_get_runtime_file always uses forward slashes regardless of platform
+  local chadrc_exists = vim.api.nvim_get_runtime_file('lua/custom/chadrc.lua', false)[1]
 
   if chadrc_exists then
+    local chadrc = require  "custom.chadrc"
```

Comment:
Try something like this instead
```suggestion
  local chadrc_path = vim.api.nvim_get_runtime_file('lua/custom/chadrc.lua', false)[1]

  if chadrc_path then
    local chadrc = dofile(chadrc_path)
```